qeynos2/Rineval_Talyas.lua

```diff
@@ -1,11 +1,12 @@
 -- items: 19071, 19070, 1000, 18302
 function event_say(e)
 	if(e.message:findi("hail")) then
-		e.self:Say(string.format("hello, %s",e.other:GetName()));
+		e.self:Say(string.format("hello, " .. e.other:GetName()));
```

Comment:
If you are going to remove the string format method you can remove the string.format call.

Reply:
Thanks, fixed!